wordpress-mobile/gutenberg-mobile
# block picker tweaks

Ref to gutenberg: https://github.com/WordPress/gutenberg/pull/23465

To test:

1. Open mobile app
2. Add `Buttons`
3. Open action sheet via an ellipsis button
4. Check if all the options within picker are working properly
5. Check if the header contains block name

PR submission checklist:
